manifests/charts/istiod-remote/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
This directory is better named `istio-remote` as to be consistent with the rest of the charts in `$ISTIO/istio/manifest`. `istiod` is an internal implementation detail, and its better that we don't expose it to users, in the unlikely event we change our architecture in the future.

Reply:
I have issue to name it Istio-remote because we will ask users to install the base charts in addition to Istiod-remote.  So it would be confusion if we call it Istio-remote.